eq_translations/survey_schema.py

```diff
@@ -120,9 +120,11 @@ def get_list_pointers(self):
         for list_pointer in list_pointers:
             schema_element = resolve_pointer(self.schema, list_pointer)
             if isinstance(schema_element, list):
-                pointers.extend(
-                    [f"{list_pointer}/{i}" for i, p in enumerate(schema_element)]
-                )
+                for i, p in enumerate(schema_element):
```

Comment:
There are existing tests for `get_list_pointers`, it would be good to add a test which ensures when you have a placeholder in the list, it is not added to the list pointers.

Reply:
Added new test in **test_survey_schemas.py**. Thanks!👍